lib/engine/game/base.rb

```diff
@@ -3208,7 +3208,7 @@ def ability_usable?(ability)
       end
 
       def player_owned_ability?(ability)
-        ability.owner&.player? || (ability.owner&.company? && ability.owner&.owner&.player?)
+        ability.owner&.player? || (!ability.owner&.corporation? && ability.owner&.owner&.player?)
```

Comment:
I think we want `(ability.owner&.company? || ability.owner&.minor?) && ability.owner&.owner&.player?` as we want to be explicit about what types of entities a player can "own".

Reply:
I was thinking that my version of the check was functionally the same, but I had forgotten about system-class entities.